scripts/index.js

```diff
@@ -1 +1,303 @@
-// Your code goes here!
\ No newline at end of file
+// Your code goes here!
+
+
+
+/* 1. Positive numbers
+ Write a function which takes an array of numbers as input and returns a new array containing only the positive numbers in the given array.
+*/
+let arr = [-1, -2, 1, 2, 3, 4]
+
+let positive = arr.filter(val => {
+    return val > 0;
+})
+//positive returns [1, 2, 3, 4]
+
+
+
+/* 2. Even Numbers
+Write a function which takes an array of numbers as input and returns a new array containing only the even numbers in the given array.
+*/
+let arr = [1, 2, 3, 4]
+
+let even = arr.filter(val => {
+    return val % 2 === 0;
+});
+//Even returns [2, 4]
+
+/* 3. Square the Numbers
+Write a function which takes an array of numbers as its argument and returns a new array containing result of squaring each of the numbers in the given array by two. Example: squareTheNumbers([1, 2, 3]) should give [1, 4, 9].
+*/
+
+var arr = [5, 7, 4, 12];
+let square = arr.map(x => x * x);
```

Comment:
Nice! Any reason you are using the implicit return here, but not with the solutions to #1 and #2?